src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -410,6 +410,7 @@ public void testPasswordRules() throws Exception {
         addUserWithPassword("ok2", "$aA123456789", HttpStatus.SC_CREATED);
         addUserWithPassword("ok3", "$Aa123456789", HttpStatus.SC_CREATED);
         addUserWithPassword("ok4", "$1aAAAAAAAAA", HttpStatus.SC_CREATED);
+        addUserWithPasswordAndHash("empty_password", "", "$%^123", HttpStatus.SC_BAD_REQUEST);
```

Comment:
Is there a test that adds user with empty password and no hash?

Reply:
Will be also good to test request to create user with `"password": null`

Reply:
> Is there a test that adds user with empty password and no hash?

Line 451-452

Reply:
Done.